image_id.go

```diff
@@ -0,0 +1,103 @@
+package flux
```

Comment:
Perhaps it's misplaced parsimony, but I'd be happier if this was all just in `image.go`.